Doc/library/zipfile.rst

```diff
@@ -838,7 +841,14 @@ Instances have the following methods and attributes:
 
    .. note::
 
-      The ZIP file format does not support timestamps before 1980.
+      The ZIP format supports multiple timestamp fields in different locations
+      (central directory, extra fields for NTFS/UNIX systems, etc.). This attribute
+      specifically returns the timestamp from the central directory. The central
+      directory timestamp format in ZIP files does not support timestamps before
+      1980. While some extra field formats (such as UNIX timestamps) can represent
+      earlier dates, this attribute only returns the central directory timestamp.
+
+      CPython interprets this timestamp as representing local time, not UTC.
```

Comment:
```suggestion
      The central directory timestamp is interpreted as representing local
      time, rather than UTC time, to match the behavior of other zip tools.
```